main/common/settings.ts

```diff
@@ -114,26 +138,27 @@ if (store.has('recordKeyboardShortcut')) {
 
 // TODO: Remove this when we feel like everyone has migrated
 if (store.has('cropperShortcut')) {
-  store.set('shortcuts.triggerCropper', shortcutToAccelerator(store.get('cropperShortcut')));
+  // TODO: Investigate type for dot notation
```

Comment:
This is not yet possible, AFAIK.

https://github.com/sindresorhus/type-fest/issues/134#issuecomment-730824008

Reply:
I thought I saw something like this on Twitter a few months ago. It might have been something that hadn't landed in a stable version yet. It's not a big deal for us, but figured we could look into it